containers/unit_tests/TestDualView.hpp

```diff
@@ -515,9 +515,12 @@ TEST(TEST_CATEGORY_DEATH, dualview_external_view_construction) {
     Kokkos::View<int*, TEST_EXECSPACE> view2("view2", 10);
 
     Kokkos::DualView<int*, TEST_EXECSPACE> v_dual(view1, view1);
+// FIXME_MSVC+CUDA error C2094: label 'gtest_label_520' was undefined
+#if !(defined(KOKKOS_COMPILER_MSVC) && defined(KOKKOS_ENABLE_CUDA))
```

Comment:
I'd rather comment the entire test out not just the ASSERT_DEATH, I mean now this is not a death test anymore ... (and this was a compilation issue right?)

Reply:
Yes, it was a compilation issue. I'm fine with just disabling the whole test.